### ISSUE TITLE
vfb_nblast is failing.

### ISSUE THREAD

**Issue by jefferis** [jefferis/vfbr#2]
```
> library(vfbr)
> vfb_nblast("fru-M-200266")
Error in vfb_nblast("fru-M-200266") : client error: (400) Bad Request
```

**Comment by jefferis** [comment]
@robbie1977 says https://virtualflybrain.slack.com/archives/D088L85PF/p1448476041000030
compare

```
 curl http://vfbdev.inf.ed.ac.uk/ocpu/library/flynblastscores/R/flycircuit_topn/csv -d 'query="FruMARCM-M002262_seg001"' 
```

with

```
curl http://vfbdev.inf.ed.ac.uk/ocpu/library/flynblastscores/R/flycircuit_topn/json -d 'query="FruMARCM-M002262_seg001"' 
```

Apparently there is a problem with the csv output!

**Comment by jefferis** [comment]
closed by 8a76b6288f2bcce9da840f2883e4a9b479ffbb10
